parlai/tasks/self_chat/worlds.py

```diff
@@ -5,51 +5,81 @@
 # LICENSE file in the root directory of this source tree.
 
 import random
+from typing import Any, Dict, List
 
+from parlai.core.agents import Agent
 from parlai.core.worlds import DialogPartnerWorld, validate
 
 
-class InteractiveWorld(DialogPartnerWorld):
+class SelfChatBaseWorld(DialogPartnerWorld):
```

Comment:
So AFAIK this was named InteractiveWorld because if you run the interactive script and set the task to `self_chat`, it automatically pulls in this module. 

@jaseweston can you comment on this?

Reply:
oh wait i see that you made it inherit from that at the bottom.

Reply:
InteractiveWorld has to be called that for interactive to work